SyncConnectionStateErrorReset/SyncConnectionStateErrorReset/SyncConnectionStateErrorResetView.swift

```diff
@@ -0,0 +1,261 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2023 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+import SwiftUI
+import RealmSwift
+
+// Entry Point
+struct InitialView: View {
+    @ObservedObject var realmManager = RealmManager()
+    @ObservedObject var app: RealmSwift.App
+    @State var isLogged: Bool = false
+
+    var body: some View {
+        // This observes changes on the App, which includes User State changes,
+        // and will trigger if the user state changes during the app cycle of the App.
+        switch app.currentUser?.state {
+        case .loggedIn:
+            if let user = app.currentUser {
+                if let realm = realmManager.realm {
+                    KioskView(user: user)
+                        .environment(\.realmConfiguration, realm.configuration)
+                        .toolbar {
+                            ToolbarItemGroup(placement: .bottomBar) {
```

Comment:
When seeing this toolbar in the running app, it wasn't obvious to me what these text strings indicated. I wonder if we should make each of these a `VStack` with something like:

```swift
VStack {
    Text("Session State")
    Text("\(realmManager.sessionState?.literal ?? "")")
}
VStack {
    Text("Connection State")
    Text("\(realmManager.connectionState?.literal ?? "")")
}
```

This would make it a little more obvious when I'm viewing the app what I'm looking at in the bottom toolbar.

If not a VStack with an individual label for each string, maybe just a `Text()` in between the two strings with something like "App State" as a reminder of what this toolbar represents?